TWLight/users/views.py

```diff
@@ -579,3 +580,109 @@ def get(self, request, pk, version, format=None):
 
         serializer = UserSerializer(users, many=True)
         return Response(serializer.data)
+
+
+class CollectionUserView(SelfOnly, ListView):
+    model = Editor
+    template_name = 'users/my_collection.html'
+
+    def get_object(self):
+        assert 'pk' in self.kwargs.keys()
+        try:
+            return Editor.objects.get(pk=self.kwargs['pk'])
+        except Editor.DoesNotExist:
+            raise Http404
+
+    def get_context_data(self, **kwargs):
+        context = super(CollectionUserView, self).get_context_data(**kwargs)
+        editor = self.get_object()
+        today = datetime.date.today()
+        proxy_bundle_authorizations = Authorization.objects.filter(Q(date_expires__gte=today) |
```

Comment:
the indentation here gets ridiculous. We should probably set some project config/hints for pycharm that we keep in version control. Not at all a blocker, just an observation.